src/elements/content-sidebar/stories/tests/BoxAISidebar-visual.stories.tsx

```diff
@@ -24,3 +24,12 @@ export const BoxAIInSidebar: StoryObj<typeof BoxAISidebar> = {
         expect(sidebar).toBeInTheDocument();
     },
 };
+
+export const ExpandButtonCheck: StoryObj<typeof BoxAISidebar> = {
```

Comment:
i assume this test will become a visual regression for the entire expanded view in Box AI right? so maybe we can name it like `BoxAIWithExpandedView`